bundle/testdata/minimal.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "mybun",
+  "version": "v1.0.0",
+  "description": "My First Bundle",
+  "invocationImages": [
+    {
+      "imageType": "docker",
+      "image": "deislabs/mybunii:def456"
+    }
+  ],
+  "images": {
+    "mybund": {
+      "imageType": "docker",
+      "image": "deislabs/mybund:abc123",
+      "description": "my bun daemon"
+    }
+  },
+  "parameters": null,
```

Comment:
Shouldn't these fields be removed 🤔 ?
I mean, should we add `omitempty` there too?

Reply:
Agreed, this one should get an omitEmpty in a follow-up PR.